doc/FEATURES_RELEASE_2.md

```diff
@@ -53,11 +54,86 @@ must display useful error messages about such constructs to the user.
 
 Visit the [homepage] and select the file
 `panacea/test/fixtures/analysis/clashes.pml`. Press `Submit`. A warning should
-now be visible outlining the `PML Construct Name Clashes` in the file.
+now be visible in the warnings tab outlining the `PML Construct Name Clashes` in
+the file.
 
 Specifically, it should highlight name clashes for constructs named `clash1`
 and `clash2`.
 
+## Identify Sequential DDIs - Complete
+
+### Description
+Sequential DDIs arise because the PML contains a workflow in which the
+dispensing of one drug is always followed by the dispensing of another drug that
+has an interaction with the first. The system should identify these sequential
+DDIs and the enclosing `sequence` construct.
```

Comment:
Sequence isn't the only possible enclosing construct for a sequential DDI. Basically anything that isn't a branch or selection is possible.